mk/travis.sh

```diff
@@ -88,7 +88,9 @@ fi
 case $TARGET_X in
 armv7-linux-androideabi)
   cargo test -vv -j2 --no-run ${mode-} ${FEATURES_X-} --target=$TARGET_X
```

Comment:
I'll add the comment.

I thought about caching the AVD but that would slow down all builds. And it will get worse when we add AArch64 Android to Travis.